Makefile

```diff
@@ -23,6 +23,12 @@ CC           = /usr/local/opt/llvm/bin/clang
 CXX          = /usr/local/opt/llvm/bin/clang++
 CGC          = $(NXDK_DIR)/tools/cg/mac/cgc
 endif
+ifneq (,$(findstring MSYS_NT,$(UNAME_S)))
+LD           = lld-link
+CC           = clang
+CXX          = clang++
+CGC          = $(NXDK_DIR)/tools/cg/win/cgc
```

Comment:
I included the 64-bit version, but I just had a thought: Perhaps I should replace it with the 32-bit version for maximum compatibility. What do you think?